src/scenarioFilter.js

```diff
@@ -0,0 +1,21 @@
+import logger from './logger';
+
+const filterScenario = (config, filter) => {
+  return new Promise(resolve => {
+    const filteredScenario = Object.values(config.scenarios).filter(
+      scenario => scenario.label === filter
+    );
+
+    if (filteredScenario.length === 0) {
+      logger.info(
+        'filterConfig',
+        `❗️ ${filter}  not found on your scenarios. Exiting Aye Spy`
+      );
+      process.exitCode = 1;
```

Comment:
I think if you throw an exception here, the run file will handle closing the app with exit code 1